packages/preview/enseeiht-internship-report/0.1.0/README.md

```diff
@@ -0,0 +1,94 @@
+# ENSEEIHT Internship Report Template
+
+Unofficial Typst Internship Report Template for ENSEEIHT
+An unofficial Typst template designed for internship reports at ENSEEIHT (École Nationale Supérieure d'Électrotechnique, d'Électronique, d'Informatique, d'Hydraulique et des Télécommunications).
+Features:
+
+Customizable cover page with school and company logos
+Automatic title formatting and numbering
+Headers and footers with integrated logos
+Support for multiple authors and supervisors
+Built-in bibliography management
+Typography optimized for French academic reports
```

Comment:
You should use Markdown syntax here to display a proper bullet list, as it is now it will display a as single paragraph without line breaks.

```suggestion
- Customizable cover page with school and company logos
- Automatic title formatting and numbering
- Headers and footers with integrated logos
- Support for multiple authors and supervisors
- Built-in bibliography management
- Typography optimized for French academic reports
```